src/app/core-components/welcome/welcome.component.html

```diff
@@ -1,99 +1,174 @@
-                                Go directly to your application portfolio to view and manage your existing assessments.
-                            </p>
-
-                        </div>
-                        <div class="d-flex justify-content-center pt-2">
-                            <button class="btn btn-primary" routerLink="/user">
-                                <fa-icon [icon]="faFolderOpen"></fa-icon>
-                                My Portfolio</button>
-                        </div>
+                    <p class="card-text fw-light">
+                        Use this tool to identify and estimate potential non-energy benefits (NEBs) that can be
+                        found while conducting energy assessments.
+                    </p>
+                </div>
+            </div>
+            <hr>
+            <div class="row">
+                <div class="col-8">
 
-                        <div class="input-group pt-3">
-                            <label>
-                                <input class="form-check-input mt-0" type="checkbox" [(ngModel)]="user.skipSplashScreen"
-                                    (change)="saveChanges()">
-                                Skip this screen next time and go directly to dashboard.</label>
-                        </div>
-                        <hr>
-                    </div>
-                    <div class="col-12 pt-3">
-                        <div class="d-flex">
-                            <fa-icon [icon]="faChevronRight"></fa-icon>
-                            <p class="card-text ms-2">
-                                Explore potential energy efficiency opportunities with Non-Energy Benefits.
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="card-title secondary">
+                                <fa-icon [icon]="faSearchPlus"></fa-icon>
+                                Find Non-Energy Benefits
+                            </div>
+                            <p class="card-text fw-light">
+                                Use the setup wizard to work through the process of finding non-energy benefits. In the
+                                wizard you will create and provide details for a company and facility. Then you will add
+                                energy assessment details that will allow you to discover additional non-energy
+                                benefits.
                             </p>
+                            <div class="d-flex justify-content-center">
+                                <button class="btn btn-outline-success" (click)="openWizardModal()">
+                                    <fa-icon [icon]="faWandMagicSparkles"></fa-icon>
+                                    Setup Wizard</button>
+                            </div>
+                            <hr>
+                            <div class="list-content">
```

Comment:
I couldn't find the class definition for list-content.

Reply:
removed